style.css.css

```diff
@@ -10,14 +10,15 @@ body {
 div {
     width: 900px;
```

Comment:
When working with CSS, it's not advisable to use px. It's better to use rem or em instead. You can do some research to see why.